jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/StepsRunner.java

```diff
@@ -497,20 +497,13 @@ private void checkImageInTargetRegistry() {
 
     results.manifestCheckResult =
         executorService.submit(
-            () -> {
-              Verify.verify(results.builtImagesAndBaseImages.get().size() == 1);
-              Future<Image> builtImage =
-                  results.builtImagesAndBaseImages.get().keySet().iterator().next();
-              Future<BlobDescriptor> containerConfigPushResult =
-                  results.builtImagesAndContainerConfigurationPushResults.get().get(builtImage);
-              return new CheckImageStep(
-                      buildContext,
-                      childProgressDispatcherFactory,
-                      results.targetRegistryClient.get(),
-                      Verify.verifyNotNull(containerConfigPushResult).get(),
-                      builtImage.get())
-                  .call();
-            });
+            () ->
+                new CheckManifestStep(
```

Comment:
Let's rename `checkImageInTargetRegistry()` to `checkManifestInTargetRegistry()` on line 494.